src/averageAges.js

```diff
@@ -15,11 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let manAvarageAge = people.filter((man) => {
+    return man.sex === 'm';
+  });
+
+  if (century) {
+    manAvarageAge = manAvarageAge.filter(man => {
+      return Math.ceil(man.died / 100) === century;
+    });
+  }
```

Comment:
So if we have century - we need to filter array twice
Let's do it only once, just choose needed condition inside filter callback